docker-compose.yml

```
@@ -72,8 +72,8 @@ services:
     volumes:
       - ./api:/api
     ports:
-      - "8000:8000"
-      - "3000:3000"
+      - "50280:8000"
+      - "50230:3000"
```

Just a small suggestion because this always confuses me when I check the port mappings in the docker compose. I never remember which to use for the Django API.

```suggestion
      # django
      - "50280:8000"
      # sphinx
      - "50230:3000"
```

---

Added comments in faa97ae.